### PR TITLE
Fix v5 subpackages

### LINKED ISSUE #176: Subpackages introspect and prop are broken with the go major module versioning
```go
conn, err := dbus.SystemBus()
if err != nil {
	return err
}
_, err = introspect.Call(conn.Object("org.freedesktop.DBus", "/"))
return err
```

Produces:

```
# command-line-arguments
./main.go:23:38: cannot use conn.Object("org.freedesktop.DBus", "/") (type "github.com/godbus/dbus/v5".BusObject) as type "github.com/godbus/dbus".BusObject in argument to introspect.Call:
        "github.com/godbus/dbus/v5".BusObject does not implement "github.com/godbus/dbus".BusObject (wrong type for Call method)
                have Call(string, "github.com/godbus/dbus/v5".Flags, ...interface {}) *"github.com/godbus/dbus/v5".Call
                want Call(string, "github.com/godbus/dbus".Flags, ...interface {}) *"github.com/godbus/dbus".Call
```

### PR DESCRIPTION
**Pull request by amenzhinsky**
Fixes #176

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -40,5 +40,5 @@ addons:
 before_install:
 
 script:
-  - go test -v -race ./...                   # Run all the tests with the race detector enabled
+  - go test -v -race -mod=readonly ./...     # Run all the tests with the race detector enabled
```
Comment (jsouthworth) [inline .travis.yml]: It looks like this doesn't work for the older releases that travis ci is running.

Comment (amenzhinsky) [reply .travis.yml]: I'd drop go1.10 because it doesn't support modules and add go1.13, what do you think on that?

Comment (jsouthworth) [reply .travis.yml]: Sounds fine to me.

Comment (amenzhinsky) [reply .travis.yml]: Pushed the change, seems like CI jobs are passing now

Comment (jsouthworth) [reply .travis.yml]: Thanks!

**Pull request merged**